fastlane/Fastfile

```diff
@@ -61,7 +61,8 @@ end
 
 desc 'Ship release build'
 lane :distribute_release_build do |_options|
-  builds = distribute_builds(release_tag: get_required_env('BUILDKITE_TAG'))
+  release_tag = get_required_env('BUILDKITE_TAG')
```

Comment:
The `release_tag` variable needs to be defined because it's used on line L69 in the call to `slack(…)` below. So defining it here will prevent the call to `slack` from crashing as it did in https://buildkite.com/automattic/studio/builds/548#018f76e3-0d68-412c-8999-1663561a6bb1/546-735